docs/pages/blog/mui-x-v6.md

```diff
@@ -0,0 +1,188 @@
+
+Add quick and customizable shortcuts for your users. Particularly useful for date ranges.
+Display them on the left, right, bottom, or top.
+
+<img src="/static/blog/v6-beta-pickers/date-range-shortcuts.png" style="width: 692px; margin-bottom: 24px;" loading="lazy" alt="Date Range shortcuts." />
+
+#### [Edit date ranges with drag and drop](/x/react-date-pickers/date-range-calendar/) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+Editing a date range is even easier now with the new drag-and-drop interface. Change `start` and `end` dates at will.
+
+<video style="margin-bottom: 24px; max-height: 398px;" autoplay muted loop playsinline >
+    <source src="/static/blog/v6-beta-pickers/edit-drag.mp4" type="video/mp4" />
+</video>
+
+#### Removed clock view on desktop Time Pickers
+
+Many users complained, and we completely agree, the clock view is not the ideal time-picking experience on Desktop, so we removed it as a default view.
+You can still use the Clock if you like to, but a replacement UI for the time picker is coming shortly after this release.
+
+## Installation and migration
+
+If this is your first try with MUI X, you can jump in the getting started section:
+
+- [Data Grid](/x/react-data-grid/getting-started/)
+- [Date Pickers](/x/react-date-pickers/getting-started/)
+
+If you're coming from previous versions, we recommend you check our migration guide with the complete list of breaking changes:
+
+- [Data Grid](/x/migration/migration-data-grid-v5/)
+- [Date Pickers](/x/migration/migration-pickers-v5/)
+
+We also provide codemods to automate some of the necessary updates in your codebase.
+
+```sh
+npx @mui/x-codemod v6.0.0/preset-safe <path>
+```
+
+## What's next?
+
+We operate in a continuous-delivery environment, so the plan is to keep rolling out new features in minor versions.
+
+With the new base established in v6.0.0, we'll continue our efforts to improve UI/UX, bring more customization abilities, and support new use cases.
+
+Here's a list of what you can expect to be delivered in the following months.
+
+### Data Grid
+
+- Clipboard importing
+- Filtering on header
+- Row Spanning
+- Improved Panels
+  - Column Management panel (column visibility, reordering, grouping, pinning, etc).
+  - Filtering panel
+- Support for Joy UI
+
+### Date Pickers
+
+- Time Picker replacement for Clock
+- Time Range Picker
+- Date Time Range Picker
+- Ability to select a month range in the Date Range Picker
+- Support for Joy UI
+
+### Charts 📊
+
+- Preview of chart components 🔥
+
+You can get more details of our next steps in [MUI X public roadmap](https://github.com/mui/mui-x/projects/1).
+
+## Decoupling versions from MUI Core
+
+We're officially decoupling MUI X's versioning from MUI Core (including `@mui/material`). We understand that this may cause confusion, so we'd like to explain the main reasons why we're moving in this direction.
```

Comment:
```suggestion
We're officially decoupling MUI X's versioning from MUI Core (including `@mui/material`: Material UI). We understand that this may cause confusion, so we'd like to explain the main reasons why we're moving in this direction.
```